example-stable.sql

```diff
@@ -0,0 +1,7 @@
+reate Table: CREATE TABLE `configdata` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `var` varchar(255) NOT NULL,
+  `val` varchar(255) NOT NULL,
+  `env` varchar(255) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8 
```

Comment:
@ncode, happy to have the example - maybe we just add it to the readme?  I'll hold off on merging if you want to update it....